pkg/brew/bfs.rb

```diff
@@ -0,0 +1,18 @@
+class Bfs < Formula
```

Comment:
They do, but from what I understand they are restrictive on what gets in. This allows us to have a tap that is easy to update and easy to use.  We can also push this to brew official if you wish and see if they want to take it.